app/javascript/packs/auth_link_clipboard.js

```diff
@@ -0,0 +1,12 @@
+import Clipboard from 'clipboard'
+
+document.addEventListener('DOMContentLoaded', () => {
+  let div = document.getElementById('clipboard-trigger-holder')
+  div.innerHTML = '<button class="btn btn-primary" id="clipboard-trigger">Copy link to clipboard</button>'
```

Comment:
Are you setting the button text here so it doesn't display if there's a problem loading the javascript? Or is there some other reason not to  include in the view HTML?

Reply:
yeah, just sort of doing progressive enhancement I guess, since it only works if there's javascript.

Reply:
👍